java/Oleksii-Riabov/src/com/zoolatech/lecture4/tasks/_7/Task7.java

```diff
@@ -0,0 +1,45 @@
+package com.zoolatech.lecture4.tasks._7;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Create a method that accepts a string of digits and after each digit
+ * appends its string representation in parentheses.
+ */
+
+public class Task7 {
+    public static void main(String[] args) {
+        String string = "12345678910";
+        System.out.println(stringRepresentation(string));
+    }
+
+    public static String stringRepresentation(String string) {
+        if (string.matches("[0-9]+") && string.length() > 2) {
+            return Stream.of(string)
+                    .map(s -> s.split("")).
```

Comment:
`s.split("")` uses a regular expression, `s.toCharArray` should be considered as a better option to split a string into individual characters

Reply:
This issue is fixed.